recursive-feature-elimination/script.whizzml

```diff
@@ -119,8 +121,31 @@
                                        "evaluation"  ev-result
                                        "last-removed" removed-fields-names})
                   evaluations (when ev (append evaluations iterations)))
-            (recur selected
-                    evaluations))))))
+              (recur selected
+                     evaluations))))))
+
+;; choose-eval-metric
+;;
+;; Sets the metric that will be use in evaluation
+;;
+;; Inputs:
+;;   evaluation-metric: (string): Evaluation metric chosen by the user
+;;                                (could be empty)
+;;   otype: (string) Objective field type
+;;
+;; Outputs: (string) Chosen evaluation-metric 
+(define (choose-eval-metric evaluation-metric otype)
+  (let (empty-eval (empty? evaluation-metric)
+        classification (= "categorical" otype)
+        eval (if empty-eval
+               (if classification
+                 "average_phi"
+                 "r_squared"
+                 evaluation-metric)))
```

Comment:
i don't think this is doing what you mean, the second `if` has 4 args; usually using `cond` instead of nested `if`s make the code more readable.  also, i wouldn't define separate variables for the conditions, they're too short:

```
eval (cond (empty? evaluation-metric) evaluation-metric
           (= "categorical" optype) "average_phi"
           "r_squared"))
```

although i am not really sure if that's what you where trying to write above (i am guessing not)